Source/bmalloc/libpas/libpas.xcodeproj/project.pbxproj

```diff
@@ -3112,7 +3120,7 @@
 		0FBAE681248EC1C000311E6F /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				CLANG_CXX_LANGUAGE_STANDARD = "gnu++14";
+				CLANG_CXX_LANGUAGE_STANDARD = "gnu++20";
```

Comment:
By convention, all build settings belong in xcconfig files. I didn't realize this was never done for libpas! No need to fix it now, but wanted to call it out.